service/src/main/java/uk/gov/hmcts/reform/fpl/model/Proceeding.java

```diff
@@ -0,0 +1,49 @@
+package uk.gov.hmcts.reform.fpl.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class Proceeding {
+    private final String onGoingProceeding;
+    private final String proceedingStatus;
+    private final String caseNumber;
+    private final String started;
```

Comment:
Should started and ended fields be of type Date?

Reply:
Yes, they should! But currently in master they exist as String. If we change type we risk loosing data.

Reply:
Ok thats fine.